sidebars.js

```diff
@@ -37,6 +37,13 @@ module.exports = {
                 "tutorials/resource-migration",
                 "tutorials/access-service-across-clusters",
                 "tutorials/workload-rebalancer",
+                {
+                    type: "category",
+                    label: "Karmadactl for Operations and Maintenance",
+                    items: [
+                        "tutorials/karmadactl/operation-scope",
```

Comment:
Are you going to demonstrate other topics of karmadactl here?
What are they?

Reply:
What I have in mind so far is:
- Multi-cluster perspective
- auto-completion